lib/hexpm_web/controllers/package_controller.ex

```diff
@@ -227,6 +231,13 @@ defmodule HexpmWeb.PackageController do
             nil
         end
     end
+    |> case do
+      nil ->
+        nil
+
+      package ->
+        hd(Packages.attach_versions([package]))
```

Comment:
Why do we need to add versions?

Reply:
My thought was that a single exact match result should include the same information as the redundant "all matches" entry which we're removing in this patch.  I'm fine with removing this line, of course—I was just working off of the earlier discussion [here](https://github.com/hexpm/hexpm/pull/1089#issuecomment-1004043266).